vitest.base.unit.config.ts

```diff
@@ -18,7 +18,9 @@ export default defineConfig({
       path.join(__dirname, "./scripts/vitest/setupFiles/customMatchers.ts"),
       path.join(__dirname, "./scripts/vitest/setupFiles/dotenv.ts"),
     ],
-    reporters: ["default", "hanging-process"],
+    reporters: process.env.GITHUB_ACTIONS
+      ? ["verbose", "hanging-process", "github-actions"]
```

Comment:
`github-actions` reporter will show the failure directly inside the PR. 

https://vitest.dev/guide/reporters.html#github-actions-reporter